hangupsbot/plugins/telesync/__init__.py

```diff
@@ -698,13 +751,180 @@ def tg_command_get_me(bot, chat_id, args):
+
+            """XXX: media sending:
+
+            * if media link is already available, send it immediately
+              * real events from google servers will have the medialink in event.conv_event.attachment
+              * media link can also be added as part of the passthru
+            * for events raised by other external chats, wait for the public link to become available
+            """
+
+            if "attachments" in event.passthru["original_request"] and event.passthru["original_request"]["attachments"]:
+                # automatically prioritise incoming events with attachments available
+                media_link = event.passthru["original_request"]["attachments"][0]
+                logger.info("media link in original request: {}".format(media_link))
+
+                yield from self._send_deferred_media(media_link, eid)
+                message = "shared media"
+                divider = ""
+
+            elif isinstance(event, FakeEvent):
+                if( "image_id" in event.passthru["original_request"]
+                        and event.passthru["original_request"]["image_id"] ):
+                    # without media link, create a deferred post until a public media link becomes available
+                    image_id = event.passthru["original_request"]["image_id"]
+                    logger.info("wait for media link: {}".format(image_id))
+
+                    loop = asyncio.get_event_loop()
+                    task = loop.create_task(
+                        self.bot._handlers.image_uri_from(
+                            image_id,
+                            self._send_deferred_media,
+                            eid ))
+
+            elif( hasattr(event, "conv_event")
+                    and hasattr(event.conv_event, "attachments")
+                    and len(event.conv_event.attachments) == 1 ):
+                # catch actual events with media link  but didn' go through the passthru
+
+                media_link = event.conv_event.attachments[0]
+                logger.info("media link in original event: {}".format(media_link))
+
+                yield from self._send_deferred_media(media_link, eid)
+                message = "shared media"
+                divider = ""
+
+            """standard message relay"""
+
+            if "sync_chat_titles" not in config or config["sync_chat_titles"] and chat_title:
+                formatted_text = "{} ({}){} {}".format( username,
+                                                        chat_title,
+                                                        divider,
+                                                        message )
+            else:
+                formatted_text = "{}{} {}".format( username,
+                                                   divider,
+                                                   message )
+
+            logger.info("sending {}: {}".format(eid, formatted_text))
+            yield from tg_bot.sendMessage( eid,
```

Comment:
since this isn't a chatbridge-specific feature (and also unhandled in original telesync), I've simply reduced the verbosity in 1725a4d so that the error will be clearer, and allowed the rest of the loop to run